app/views/change_logs/_log_item.html.slim

```diff
@@ -1,3 +1,4 @@
-.change-log-item data-change-log-role="item"
-  h2.govuk-heading-s = Date.parse(change_log[:change_on]).strftime("%-d %B %Y")
+.change-log-item class="govuk-!-margin-bottom-7"
```

Comment:
Did we not get rid of this `change-log-item` class?

Reply:
Good spot! Yes, that's now entirely redundant. I'll remove it.